src/new/wallet.js

```diff
@@ -1122,6 +1131,21 @@ class HathorWallet extends EventEmitter {
     return history;
   }
 
+  /**
+   * Process the transactions on the websocket transaction queue as if they just arrived.
+   *
+   * @memberof HathorWallet
+   * @inner
+   */
+  async processTxQueue() {
+    let wsData = this.wsTxQueue.dequeue();
+    while(wsData !== undefined) {
+      // process wsData like it just arrived
+      this.onNewTx(wsData);
+      wsData = this.wsTxQueue.dequeue();
```

Comment:
Should we process one transaction per call and just schedule this method to run again in zero seconds? This would unblock the main event loop to handle other stuff if needed.

Reply:
Done.

Reply:
Great solution! Thanks!